src/Service/Indexing/IndexingOrganization.php

```diff
@@ -0,0 +1,107 @@
+    }
+
+    public function serialize(IndexItemInterface $item): array
+    {
+        $contextBuilder = (new ObjectNormalizerContextBuilder())
+            ->withGroups([IndexNames::Organization->value]);
+        $contextBuilder = (new DateTimeNormalizerContextBuilder())
+            ->withContext($contextBuilder)
+            ->withFormat(IndexFieldTypes::DATEFORMAT);
+
+        return $this->serializer->normalize($item, null, $contextBuilder->toArray());
+    }
+
+    /**
+     * @throws IndexingException
+     */
+    protected function createEsIndex(string $indexName): void
+    {
+        try {
+            /** @var Elasticsearch $response */
+            $response = $this->client->indices()->create([
+                'index' => $indexName,
+                'body' => [
+                    'settings' => [
+                        'number_of_shards' => 5,
+                        'number_of_replicas' => 0,
+                    ],
+                    'mappings' => [
+                        'dynamic' => 'strict',
+                        'properties' => [
+                            'entityId' => [
+                                'type' => 'integer',
+                                'doc_values' => false,
+                            ],
+                            'name' => [
+                                'type' => 'keyword',
+                                'index_options' => 'docs',
+                                'index' => false,
+                                'doc_values' => false,
+                                'norms' => false,
+                            ],
+                            'email' => [
+                                'type' => 'keyword',
+                                'index_options' => 'docs',
+                                'index' => false,
+                                'doc_values' => false,
+                                'norms' => false,
+                            ],
+                            'url' => [
+                                'type' => 'keyword',
+                                'index_options' => 'docs',
+                                'index' => false,
+                                'doc_values' => false,
+                                'norms' => false,
+                            ],
+                            'created' => [
+                                'type' => 'date',
+                                'format' => IndexFieldTypes::DATEFORMAT_ES,
+                                'index' => false,
+                                'doc_values' => true,
+                            ],
+                            'updated' => [
+                                'type' => 'date',
+                                'format' => IndexFieldTypes::DATEFORMAT_ES,
+                                'index' => false,
+                                'doc_values' => true,
+                            ],
+                        ],
+                    ],
+                ],
```

Comment:
Should/can we move this to config in `services.yaml`, say?

Reply:
I like the idea, but most of this is searchable at ES in php... so this in php matches the docs at ES.